### ISSUE TITLE
📃: Cassava Leaf Disease Classification Model

### ISSUE THREAD

**Issue by Avdhesh-Varshney** [Avdhesh-Varshney/AI-Code#40]
:red_circle: **Title** : Cassava Leaf Disease Classification Model
:red_circle: **Aim** : Brief approach for this project from a basic level upto highest possible accuracy.
:red_circle: **Brief Explanation** : 
- Link of the dataset
- Perfoming EDA
- Attach any blog for reading related to the topic
- Explain the features properly
- Add necessary comments for the code understanding
- Attach your notebook links
- Applying atleast 3-4 models and compairing them.

**NOTE:** *Follow this [readme template](https://github.com/Avdhesh-Varshney/AI-Code/blob/main/PROJECT-README-TEMPLATE.md)*

### Screenshots 📷

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- What is your participant role? (Mention the Open Source Program name. Eg. GSOC, GSSOC, SSOC, JWOC, etc.)

***********************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Armaan457** [comment]
Sir what dataset is to be used here?

**Comment by Avdhesh-Varshney** [comment]
> Sir what dataset is to be used here?

It's totally dependent to the contributors.

**Comment by Nithish-456** [comment]
Full name : Nithish Paidimarri
What is your participant role? SSOC'24
Dataset: https://www.kaggle.com/datasets/gauravduttakiit/cassava-leaf-disease-classification/data
Approach: I will implement this project with different cnn models like own CNN architecture, VGG16, ResNet101, InceptionNetV3 and performing hyperparameter tuning to enhance the model accuracy.

@Avdhesh-Varshney Please assign to me if above things are fine!
Thanks

**Comment by Avdhesh-Varshney** [comment]
@Nithish-456 go through the guidelines once before raising the pr.

**Comment by shivansh-2003** [comment]
Full name : Shivansh Mahajan
What is your participant role? SWOC 24
I will  custom convolutional neural network for image classification.

**Comment by Avdhesh-Varshney** [comment]
@shivansh-2003 What's the update?
